sdk/servicebus/ci.yml

```diff
@@ -4,6 +4,7 @@ trigger:
   branches:
     include:
     - master
+      - main
```

Comment:
The indents look off to me.  Did you intend `main` to be a child of `master`?    Leaving one comment here, but this seems to apply across the changeset.

Reply:
Good catch. I have to adjust a bit. Thanks!

Reply:
Fixed